CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Team and configuration owners
+peribolos.yaml       @afflom @jpower432 @sabre1041 @usrbinkat
+
+# organizational docs owners
+profile/* @emporous/docs-maintainers
```

Comment:
`docs-maintainers` is nested underneath `docs`. May need to specify the parent too

Reply:
I meant to add a blurb about this in the description, but CODEOWNERS is showing invalid because the `docs-maintainers` team doesn't have write access to the repo yet. Added with this PR.